include/exec/any_sender_of.hpp

```diff
@@ -1060,10 +1075,6 @@ namespace exec {
             __storage_.__get_object_pointer(), static_cast<__receiver_ref_t&&>(__receiver));
         }
 
-        explicit operator bool() const noexcept {
-          return __get_object_pointer(__storage_) != nullptr;
-        }
-
```

Comment:
i'm curious why you removed this conversion operator.

Reply:
it's just a drive-by cleanup, can remove it. from corresponding commit-message (i should have put it in the MR-text as well):

Remove unnused operator bool on `any_sender` base.

Not used anywhere and implementation calls a function that does
not exist: `__get_object_pointer(__storage)`. Instead it could
use `__storage_.__get_object_pointer()`, but since this is dead
code, might as well remove it.